inference/server/oasst_inference_server/settings.py

```diff
@@ -44,6 +45,9 @@ def allowed_model_config_names_list(self) -> list[str]:
     postgres_password: str = "postgres"
     postgres_db: str = "postgres"
 
+    insert_fake_data: bool = True
+    fake_data_path: Optional[FilePath] = "/tmp/test_data/realistic/realistic_seed_data.json"
```

Comment:
We should use `FilePath | None` and then we can avoid the need for `Optional` import